### PR TITLE
feat(node-workspace): Allow to specify version prefix in config

### PR DESCRIPTION
**Pull request by danez**
This allows a `versionPrefix` config option that can be set to a string (like `~`, `^`, ` `, `>=`) that will be used when updating linked local dependencies in a package.

- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/release-please/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: docs/manifest-releaser.md [hunk 1]
```diff
@@ -172,6 +172,10 @@ defaults (those are documented in comments)
   // absence defaults to false
   "bump-patch-for-minor-pre-major": true,
 
+  // prefix linked package version updates with this string
+  // absence defaults to `^`
+  "version-prefix": true,
```
Comment (chingor13) [inline docs/manifest-releaser.md:177]:
```suggestion
  "version-prefix": "^",
```

**Review by chingor13 — CHANGES_REQUESTED** [review]
Thanks for this pull request and sorry for the delay in the review.

I'm torn on whether this option should be global (like it's implemented here) or should be localized to the `node-workspace` plugin (and potentially other workspace plugins). The reason being that these version prefixes only really apply to dependency declarations (which are handled by workspace plugins).

If localized to the `node-workspace` plugin, we would add a config like:

```json
{
  "plugins": [
    {
      "type": "node-workspace",
      "versionPrefix": "~"
    }
  ]
}
```

and then have the version prefix as a constructor argument to the workspace plugins.

What do you think?

**Comment by chingor13** [comment]
Another option too, is to have the workspace plugins be smarter when updating the dependencies and preserve whatever prefix is currently being used.

**Comment by danez** [comment]
> Another option too, is to have the workspace plugins be smarter when updating the dependencies and preserve whatever prefix is currently being used.

I actually like this the best, will try to find some time to look into this.

**Pull request closed without merging**